mod/reportedcontent/languages/en.php

```diff
@@ -21,7 +21,6 @@
 	'reportedcontent:description' => 'Why are you reporting this?',
 	'reportedcontent:address' => 'Location of the item',
 	'reportedcontent:success' => 'Your report has been sent to the site admin',
-	'reportedcontent:report' => 'Report this',
```

Comment:
Why removing this language string? It's still in use, isn't it?

Reply:
Ah, duplicate! I was confused by the seemingly missing connection to the "update composer dependencies" title of the PR.